py/server/deephaven/table.py

```diff
@@ -330,6 +352,7 @@ def update(self, formulas: Union[str, Sequence[str]]) -> Table:
         """
         try:
             formulas = to_sequence(formulas)
+            _set_query_scope()
```

Comment:
Don't we need this in a lot more places? All of `view`, `select`, `updateView`, `update`, `lazyUpdate`, `where`, `selectDistinct`, at a minimum.